functions/src/backup-db.ts

```diff
@@ -18,46 +18,62 @@
 
 import * as functions from 'firebase-functions'
 import * as firestore from '@google-cloud/firestore'
-const client = new firestore.v1.FirestoreAdminClient()
+import { FirestoreAdminClient } from '@google-cloud/firestore/types/v1/firestore_admin_client'
 
-const bucket = 'gs://manifold-firestore-backup'
+export const backupDbCore = async (
+  client: FirestoreAdminClient,
+  project: string,
+  bucket: string
+) => {
+  const name = client.databasePath(project, '(default)')
+  const outputUriPrefix = `gs://${bucket}`
+  // Leave collectionIds empty to export all collections
+  // or set to a list of collection IDs to export,
+  // collectionIds: ['users', 'posts']
+  // NOTE: Subcollections are not backed up by default
+  const collectionIds = [
+    'contracts',
+    'groups',
+    'private-users',
+    'stripe-transactions',
+    'transactions',
+    'users',
+    'bets',
+    'comments',
+    'followers',
```

Comment:
Add `follows` which is a subcollection of users. (Whereas `followers` is a subcollection of folds).